static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-media.scss

```diff
@@ -1,41 +1,16 @@
-.fc-item--type-media {
-    &.fc-item--audio {
-        .inline-volume-high {
-            position: relative;
-            top: 2px;
-
-            > svg {
-                width: 18px;
-            }
-        }
-
-    }
-
-    &.fc-item--gallery {
-        .inline-camera {
-            position: relative;
-            top: 2px;
-
-            > svg {
-                width: 18px;
-            }
-        }
-    }
-}
-
 // Styles shared by media and interview (video) types
 .fc-item--type-interview.fc-item--video,
 .fc-item--type-media {
     background-color: $media-garnett-main-1;
 
-    .inline-icon__svg:not(.inline-clock__svg) {
-        padding: 5px;
-        border-radius: 50%;
-    }
+    // .inline-icon__svg:not(.inline-clock__svg) {
+    //     padding: 5px;
+    //     border-radius: 50%;
+    // }
```

Comment:
Is it safe to delete this?

Reply:
ahh yeah, this was the initial problem, i'll delete it!